pages/common/from.md

```diff
@@ -0,0 +1,19 @@
+# from
+
+> Prints mail header lines from your mailbox.
+
+- List mail:
+
+`from`
+
+- Print count of messages and exit:
```

Comment:
`and exit` might be irrelevant here to better align with other arguments.

What do you think?

Reply:
Good idea.  I was quoting directly from the man page:
>
>      -c      Just print a count of messages and exit.